config/migrations/2024/20241108111400-change-rangorde-GR-to-lid.sparql

```diff
@@ -0,0 +1,26 @@
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX org: <http://www.w3.org/ns/org#>
+
+DELETE {
+  GRAPH ?graph {
+  ?mandataris mandaat:rangorde ?rangorde.
+  }
+}
+INSERT {
+  GRAPH ?graph {
+    ?mandataris mandaat:rangorde ?newLabel.
+  }
+}
+WHERE {
```

Comment:
this query didn't work for me when running on a prod backup, there are still cases of "Gemeenteraadslid" in the rangorde data. Verify with:
```
PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
SELECT * WHERE {
  GRAPH ?g {
  ?s mandaat:rangorde ?rank.
  FILTER(CONTAINS(?rank, "Gemeenteraadslid"))
  }
} 
```

Reply:
The mandatarissen that you get returned here for "Gemeenteraadslid" do not have Gemeenteraad classificatie.. The migration works for the classificatie for Gemeenteraad

These are the bestuursfunctie codes:
- https://data.vlaanderen.be/doc/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000012 **Voorzitter van de gemeenteraad**
- https://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000015 **Lid van de Raad voor Maatschappelijk Welzijn**

Reply:
As above

Reply:
You can check if all items are replaced by just running the query with a select

```
SELECT ?rangorde ?newLabel
```